### PR TITLE
#1123: Replace composer install with composer require in composer_not_command rule

### LINKED ISSUE #1123: No suggestions for incorrectly using composer install instead of composer require
Maybe this is just me, but often when I want to install a new package with composer I'll try `composer install package` when I should be using `composer require package`. Composer prints the message "Invalid argument package. Use "composer require package" instead to add packages to your composer.json." when you do this, so I expected thefuck to pick up on that.

The output of `thefuck --version` (something like `The Fuck 3.1 using Python
3.5.0 and Bash 4.4.12(1)-release`):

    The Fuck 3.30 using Python 3.8.2 and Bash 5.0.17(1)-release

Your system (Debian 7, ArchLinux, Windows, etc.):

    Ubuntu 20.04

How to reproduce the bug:

```shell
$ composer install package
Invalid argument package. Use "composer require package" instead to add packages to your composer.json.
$ fuck
Nothing found
```

The output of The Fuck with `THEFUCK_DEBUG=true` exported (typically execute `export THEFUCK_DEBUG=true` in your shell before The Fuck):
```
DEBUG: Run with settings: {'alter_history': True,
 'debug': True,
 'env': {'GIT_TRACE': '1', 'LANG': 'C', 'LC_ALL': 'C'},
 'exclude_rules': [],
 'history_limit': None,
 'instant_mode': False,
 'no_colors': False,
 'num_close_matches': 3,
 'priority': {},
 'repeat': False,
 'require_confirmation': True,
 'rules': [<const: All rules enabled>],
 'slow_commands': ['lein', 'react-native', 'gradle', './gradlew', 'vagrant'],
 'user_dir': PosixPath('/home/workini/.config/thefuck'),
 'wait_command': 3,
 'wait_slow_command': 15}
DEBUG: Received output: Invalid argument package. Use "composer require package" instead to add packages to your composer.json.

DEBUG: Call: composer install package; with env: {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/REDSTONEBLOCK:@/tmp/.ICE-unix/3333,unix/REDSTONEBLOCK:/tmp/.ICE-unix/3333', 'QT_ACCESSIBILITY': '1', 'TF_SHELL': 'bash', 'COLORTERM': 'truecolor', 'XDG_CONFIG_DIRS': '/etc/xdg/xdg-ubuntu:/etc/xdg', 'NVM_INC': '/home/workini/.nvm/versions/node/v12.18.3/include/node', 'XDG_MENU_PREFIX': 'gnome-', 'GNOME_DESKTOP_SESSION_ID': 'this-is-deprecated', 'GTK_IM_MODULE': 'ibus', 'LANGUAGE': 'en_GB:en', 'QT4_IM_MODULE': 'ibus', 'GNOME_SHELL_SESSION_MODE': 'ubuntu', 'SSH_AUTH_SOCK': '/run/user/1001/keyring/ssh', 'XMODIFIERS': '@im=ibus', 'DESKTOP_SESSION': 'ubuntu', 'SSH_AGENT_PID': '3219', 'GTK_MODULES': 'gail:atk-bridge', 'PWD': '/home/workini/Code/avro-energy/portal', 'LOGNAME': 'workini', 'XDG_SESSION_DESKTOP': 'ubuntu', 'XDG_SESSION_TYPE': 'x11', 'GPG_AGENT_INFO': '/run/user/1001/gnupg/S.gpg-agent:0:1', 'XAUTHORITY': '/run/user/1001/gdm/Xauthority', 'GJS_DEBUG_TOPICS': 'JS ERROR;JS LOG', 'WINDOWPATH': '2', 'HOME': '/home/workini', 'USERNAME': 'workini', 'IM_CONFIG_PHASE': '1', 'LANG': 'C', 'LS_COLORS': 'rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:', 'XDG_CURRENT_DESKTOP': 'ubuntu:GNOME', 'VTE_VERSION': '6003', 'GNOME_TERMINAL_SCREEN': '/org/gnome/Terminal/screen/5a99e198_deb1_4832_9908_2527cbbed078', 'INVOCATION_ID': '4714ae1119714268adadf185d5163e23', 'MANAGERPID': '3014', 'CLUTTER_IM_MODULE': 'ibus', 'TF_ALIAS': 'f', 'GJS_DEBUG_OUTPUT': 'stderr', 'NVM_DIR': '/home/workini/.nvm', 'LESSCLOSE': '/usr/bin/lesspipe %s %s', 'XDG_SESSION_CLASS': 'user', 'TERM': 'xterm-256color', 'TF_HISTORY': '\t thefuck --version | clip\n\t osname\n\t uname\n\t tldr uname\n\t uname -s\n\t uname -srv\n\t composer install package\n\t f\n\t export THEFUCK_DEBUG=true\n\t composer install package', 'LESSOPEN': '| /usr/bin/lesspipe %s', 'USER': 'workini', 'PYTHONIOENCODING': 'utf-8', 'TF_SHELL_ALIASES': 'alias alert=\'notify-send --urgency=low -i "$([ $? = 0 ] && echo terminal || echo error)" "$(history|tail -n1|sed -e \'\\\'\'s/^\\s*[0-9]\\+\\s*//;s/[;&|]\\s*alert$//\'\\\'\')"\'\nalias clip=\'xclip -sel clip\'\nalias egrep=\'egrep --color=auto\'\nalias fgrep=\'fgrep --color=auto\'\nalias fin=\'printf \'\\\'\'\\a\'\\\'\'\'\nalias grep=\'grep --color=auto\'\nalias hartisan=\'homestead artisan\'\nalias hphpunit=\'homestead cmd vendor/bin/phpunit\'\nalias htinker=\'hartisan tinker\'\nalias l=\'ls -CF\'\nalias la=\'ls -A\'\nalias ll=\'ls -alF\'\nalias ls=\'ls --color=auto\'\nalias phpunit=\'vendor/bin/phpunit\'\nalias tinker=\'artisan tinker\'', 'GNOME_TERMINAL_SERVICE': ':1.121', 'DISPLAY': ':0', 'SHLVL': '1', 'NVM_CD_FLAGS': '', 'QT_IM_MODULE': 'ibus', 'XDG_RUNTIME_DIR': '/run/user/1001', 'JOURNAL_STREAM': '9:58545', 'XDG_DATA_DIRS': '/usr/share/ubuntu:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop', 'PATH': '/home/workini/.nvm/versions/node/v12.18.3/bin:/home/workini/.local/bin:/home/workini/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'GDMSESSION': 'ubuntu', 'THEFUCK_DEBUG': 'true', 'DBUS_SESSION_BUS_ADDRESS': 'unix:path=/run/user/1001/bus', 'NVM_BIN': '/home/workini/.nvm/versions/node/v12.18.3/bin', 'OLDPWD': '/home/workini', '_': '/usr/local/bin/thefuck', 'LC_ALL': 'C', 'GIT_TRACE': '1'}; is slow: False took: 0:00:00.093402
DEBUG: Importing rule: adb_unknown_command; took: 0:00:00.000417
DEBUG: Importing rule: ag_literal; took: 0:00:00.000760
DEBUG: Importing rule: apt_get; took: 0:00:00.014360
DEBUG: Importing rule: apt_get_search; took: 0:00:00.000385
DEBUG: Importing rule: apt_invalid_operation; took: 0:00:00.000862
DEBUG: Importing rule: apt_list_upgradable; took: 0:00:00.000462
DEBUG: Importing rule: apt_upgrade; took: 0:00:00.000428
DEBUG: Importing rule: aws_cli; took: 0:00:00.000270
DEBUG: Importing rule: az_cli; took: 0:00:00.000269
DEBUG: Importing rule: brew_cask_dependency; took: 0:00:00.000659
DEBUG: Importing rule: brew_install; took: 0:00:00.000111
DEBUG: Importing rule: brew_link; took: 0:00:00.000266
DEBUG: Importing rule: brew_reinstall; took: 0:00:00.000556
DEBUG: Importing rule: brew_uninstall; took: 0:00:00.000274
DEBUG: Importing rule: brew_unknown_command; took: 0:00:00.000128
DEBUG: Importing rule: brew_update_formula; took: 0:00:00.000275
DEBUG: Importing rule: brew_upgrade; took: 0:00:00.000107
DEBUG: Importing rule: cargo; took: 0:00:00.000093
DEBUG: Importing rule: cargo_no_command; took: 0:00:00.000272
DEBUG: Importing rule: cat_dir; took: 0:00:00.000268
DEBUG: Importing rule: cd_correction; took: 0:00:00.001175
DEBUG: Importing rule: cd_mkdir; took: 0:00:00.000426
DEBUG: Importing rule: cd_parent; took: 0:00:00.000098
DEBUG: Importing rule: chmod_x; took: 0:00:00.000101
DEBUG: Importing rule: choco_install; took: 0:00:00.000470
DEBUG: Importing rule: composer_not_command; took: 0:00:00.000280
DEBUG: Importing rule: cp_create_destination; took: 0:00:00.000266
DEBUG: Importing rule: cp_omitting_directory; took: 0:00:00.000425
DEBUG: Importing rule: cpp11; took: 0:00:00.000341
DEBUG: Importing rule: dirty_untar; took: 0:00:00.001123
DEBUG: Importing rule: dirty_unzip; took: 0:00:00.001456
DEBUG: Importing rule: django_south_ghost; took: 0:00:00.000111
DEBUG: Importing rule: django_south_merge; took: 0:00:00.000098
DEBUG: Importing rule: dnf_no_such_command; took: 0:00:00.000839
DEBUG: Importing rule: docker_image_being_used_by_container; took: 0:00:00.000271
DEBUG: Importing rule: docker_login; took: 0:00:00.000266
DEBUG: Importing rule: docker_not_command; took: 0:00:00.000605
DEBUG: Importing rule: dry; took: 0:00:00.000098
DEBUG: Importing rule: fab_command_not_found; took: 0:00:00.000484
DEBUG: Importing rule: fix_alt_space; took: 0:00:00.000279
DEBUG: Importing rule: fix_file; took: 0:00:00.002043
DEBUG: Importing rule: gem_unknown_command; took: 0:00:00.000456
DEBUG: Importing rule: git_add; took: 0:00:00.000492
DEBUG: Importing rule: git_add_force; took: 0:00:00.000265
DEBUG: Importing rule: git_bisect_usage; took: 0:00:00.000261
DEBUG: Importing rule: git_branch_delete; took: 0:00:00.000269
DEBUG: Importing rule: git_branch_delete_checked_out; took: 0:00:00.000267
DEBUG: Importing rule: git_branch_exists; took: 0:00:00.000337
DEBUG: Importing rule: git_branch_list; took: 0:00:00.000255
DEBUG: Importing rule: git_checkout; took: 0:00:00.000267
DEBUG: Importing rule: git_commit_amend; took: 0:00:00.000260
DEBUG: Importing rule: git_commit_reset; took: 0:00:00.000254
DEBUG: Importing rule: git_diff_no_index; took: 0:00:00.000253
DEBUG: Importing rule: git_diff_staged; took: 0:00:00.000319
DEBUG: Importing rule: git_fix_stash; took: 0:00:00.000270
DEBUG: Importing rule: git_flag_after_filename; took: 0:00:00.000262
DEBUG: Importing rule: git_help_aliased; took: 0:00:00.000252
DEBUG: Importing rule: git_merge; took: 0:00:00.000313
DEBUG: Importing rule: git_merge_unrelated; took: 0:00:00.000258
DEBUG: Importing rule: git_not_command; took: 0:00:00.000259
DEBUG: Importing rule: git_pull; took: 0:00:00.000264
DEBUG: Importing rule: git_pull_clone; took: 0:00:00.000251
DEBUG: Importing rule: git_pull_uncommitted_changes; took: 0:00:00.000254
DEBUG: Importing rule: git_push; took: 0:00:00.000273
DEBUG: Importing rule: git_push_different_branch_names; took: 0:00:00.000250
DEBUG: Importing rule: git_push_force; took: 0:00:00.000253
DEBUG: Importing rule: git_push_pull; took: 0:00:00.000265
DEBUG: Importing rule: git_push_without_commits; took: 0:00:00.000320
DEBUG: Importing rule: git_rebase_merge_dir; took: 0:00:00.000257
DEBUG: Importing rule: git_rebase_no_changes; took: 0:00:00.000183
DEBUG: Importing rule: git_remote_delete; took: 0:00:00.000260
DEBUG: Importing rule: git_remote_seturl_add; took: 0:00:00.000180
DEBUG: Importing rule: git_rm_local_modifications; took: 0:00:00.000255
DEBUG: Importing rule: git_rm_recursive; took: 0:00:00.000256
DEBUG: Importing rule: git_rm_staged; took: 0:00:00.000252
DEBUG: Importing rule: git_stash; took: 0:00:00.000252
DEBUG: Importing rule: git_stash_pop; took: 0:00:00.000315
DEBUG: Importing rule: git_tag_force; took: 0:00:00.000268
DEBUG: Importing rule: git_two_dashes; took: 0:00:00.000257
DEBUG: Importing rule: go_run; took: 0:00:00.000269
DEBUG: Importing rule: go_unknown_command; took: 0:00:00.000389
DEBUG: Importing rule: gradle_no_task; took: 0:00:00.000498
DEBUG: Importing rule: gradle_wrapper; took: 0:00:00.000271
DEBUG: Importing rule: grep_arguments_order; took: 0:00:00.000302
DEBUG: Importing rule: grep_recursive; took: 0:00:00.000262
DEBUG: Importing rule: grunt_task_not_found; took: 0:00:00.000470
DEBUG: Importing rule: gulp_not_task; took: 0:00:00.000278
DEBUG: Importing rule: has_exists_script; took: 0:00:00.000265
DEBUG: Importing rule: heroku_multiple_apps; took: 0:00:00.000271
DEBUG: Importing rule: heroku_not_command; took: 0:00:00.000268
DEBUG: Importing rule: history; took: 0:00:00.000099
DEBUG: Importing rule: hostscli; took: 0:00:00.000425
DEBUG: Importing rule: ifconfig_device_not_found; took: 0:00:00.000333
DEBUG: Importing rule: java; took: 0:00:00.000315
DEBUG: Importing rule: javac; took: 0:00:00.000259
DEBUG: Importing rule: lein_not_task; took: 0:00:00.000413
DEBUG: Importing rule: ln_no_hard_link; took: 0:00:00.000257
DEBUG: Importing rule: ln_s_order; took: 0:00:00.000262
DEBUG: Importing rule: long_form_help; took: 0:00:00.000102
DEBUG: Importing rule: ls_all; took: 0:00:00.000259
DEBUG: Importing rule: ls_lah; took: 0:00:00.000265
DEBUG: Importing rule: man; took: 0:00:00.000272
DEBUG: Importing rule: man_no_space; took: 0:00:00.000097
DEBUG: Importing rule: mercurial; took: 0:00:00.000265
DEBUG: Importing rule: missing_space_before_subcommand; took: 0:00:00.000109
DEBUG: Importing rule: mkdir_p; took: 0:00:00.000257
DEBUG: Importing rule: mvn_no_command; took: 0:00:00.000269
DEBUG: Importing rule: mvn_unknown_lifecycle_phase; took: 0:00:00.000271
DEBUG: Importing rule: nixos_cmd_not_found; took: 0:00:00.000451
DEBUG: Importing rule: no_command; took: 0:00:00.000284
DEBUG: Importing rule: no_such_file; took: 0:00:00.000105
DEBUG: Importing rule: npm_missing_script; took: 0:00:00.000562
DEBUG: Importing rule: npm_run_script; took: 0:00:00.000273
DEBUG: Importing rule: npm_wrong_command; took: 0:00:00.000518
DEBUG: Importing rule: open; took: 0:00:00.000423
DEBUG: Importing rule: pacman; took: 0:00:00.000483
DEBUG: Importing rule: pacman_not_found; took: 0:00:00.000108
DEBUG: Importing rule: path_from_history; took: 0:00:00.000118
DEBUG: Importing rule: php_s; took: 0:00:00.000272
DEBUG: Importing rule: pip_install; took: 0:00:00.000349
DEBUG: Importing rule: pip_unknown_command; took: 0:00:00.000350
DEBUG: Importing rule: port_already_in_use; took: 0:00:00.000202
DEBUG: Importing rule: prove_recursively; took: 0:00:00.000286
DEBUG: Importing rule: pyenv_no_such_command; took: 0:00:00.000529
DEBUG: Importing rule: python_command; took: 0:00:00.000259
DEBUG: Importing rule: python_execute; took: 0:00:00.000272
DEBUG: Importing rule: quotation_marks; took: 0:00:00.000095
DEBUG: Importing rule: react_native_command_unrecognized; took: 0:00:00.000347
DEBUG: Importing rule: remove_shell_prompt_literal; took: 0:00:00.000097
DEBUG: Importing rule: remove_trailing_cedilla; took: 0:00:00.000096
DEBUG: Importing rule: rm_dir; took: 0:00:00.000262
DEBUG: Importing rule: rm_root; took: 0:00:00.000279
DEBUG: Importing rule: scm_correction; took: 0:00:00.000280
DEBUG: Importing rule: sed_unterminated_s; took: 0:00:00.000443
DEBUG: Importing rule: sl_ls; took: 0:00:00.000097
DEBUG: Importing rule: ssh_known_hosts; took: 0:00:00.000279
DEBUG: Importing rule: sudo; took: 0:00:00.000109
DEBUG: Importing rule: sudo_command_from_user_path; took: 0:00:00.000279
DEBUG: Importing rule: switch_lang; took: 0:00:00.000162
DEBUG: Importing rule: systemctl; took: 0:00:00.000431
DEBUG: Importing rule: terraform_init; took: 0:00:00.000277
DEBUG: Importing rule: test.py; took: 0:00:00.000094
DEBUG: Importing rule: tmux; took: 0:00:00.000264
DEBUG: Importing rule: touch; took: 0:00:00.000267
DEBUG: Importing rule: tsuru_login; took: 0:00:00.000266
DEBUG: Importing rule: tsuru_not_command; took: 0:00:00.000263
DEBUG: Importing rule: unknown_command; took: 0:00:00.000100
DEBUG: Importing rule: unsudo; took: 0:00:00.000093
DEBUG: Importing rule: vagrant_up; took: 0:00:00.000268
DEBUG: Importing rule: whois; took: 0:00:00.000401
DEBUG: Importing rule: workon_doesnt_exists; took: 0:00:00.000348
DEBUG: Importing rule: yarn_alias; took: 0:00:00.000264
DEBUG: Importing rule: yarn_command_not_found; took: 0:00:00.000577
DEBUG: Importing rule: yarn_command_replaced; took: 0:00:00.000421
DEBUG: Importing rule: yarn_help; took: 0:00:00.000269
DEBUG: Importing rule: yum_invalid_operation; took: 0:00:00.000642
DEBUG: Trying rule: path_from_history; took: 0:00:00.000445
DEBUG: Trying rule: dry; took: 0:00:00.000070
DEBUG: Trying rule: git_stash_pop; took: 0:00:00.000019
DEBUG: Trying rule: test.py; took: 0:00:00.000002
DEBUG: Trying rule: adb_unknown_command; took: 0:00:00.000013
DEBUG: Trying rule: ag_literal; took: 0:00:00.000013
DEBUG: Trying rule: apt_get; took: 0:00:00.000003
DEBUG: Trying rule: apt_get_search; took: 0:00:00.000013
DEBUG: Trying rule: apt_invalid_operation; took: 0:00:00.000015
DEBUG: Trying rule: apt_list_upgradable; took: 0:00:00.000014
DEBUG: Trying rule: apt_upgrade; took: 0:00:00.000012
DEBUG: Trying rule: aws_cli; took: 0:00:00.000012
DEBUG: Trying rule: az_cli; took: 0:00:00.000012
DEBUG: Trying rule: brew_link; took: 0:00:00.000013
DEBUG: Trying rule: brew_reinstall; took: 0:00:00.000011
DEBUG: Trying rule: brew_uninstall; took: 0:00:00.000011
DEBUG: Trying rule: brew_update_formula; took: 0:00:00.000014
DEBUG: Trying rule: cargo; took: 0:00:00.000002
DEBUG: Trying rule: cargo_no_command; took: 0:00:00.000012
DEBUG: Trying rule: cat_dir; took: 0:00:00.000012
DEBUG: Trying rule: cd_correction; took: 0:00:00.000013
DEBUG: Trying rule: cd_mkdir; took: 0:00:00.000011
DEBUG: Trying rule: cd_parent; took: 0:00:00.000002
DEBUG: Trying rule: chmod_x; took: 0:00:00.000003
DEBUG: Trying rule: composer_not_command; took: 0:00:00.000014
DEBUG: Trying rule: cp_create_destination; took: 0:00:00.000012
DEBUG: Trying rule: cp_omitting_directory; took: 0:00:00.000012
DEBUG: Trying rule: cpp11; took: 0:00:00.000011
DEBUG: Trying rule: dirty_untar; took: 0:00:00.000012
DEBUG: Trying rule: dirty_unzip; took: 0:00:00.000012
DEBUG: Trying rule: django_south_ghost; took: 0:00:00.000003
DEBUG: Trying rule: django_south_merge; took: 0:00:00.000002
DEBUG: Trying rule: docker_image_being_used_by_container; took: 0:00:00.000012
DEBUG: Trying rule: docker_login; took: 0:00:00.000010
DEBUG: Trying rule: docker_not_command; took: 0:00:00.000012
DEBUG: Trying rule: fab_command_not_found; took: 0:00:00.000012
DEBUG: Trying rule: fix_alt_space; took: 0:00:00.000003
DEBUG: Trying rule: fix_file; took: 0:00:00.000007
DEBUG: Trying rule: gem_unknown_command; took: 0:00:00.000012
DEBUG: Trying rule: git_add; took: 0:00:00.000010
DEBUG: Trying rule: git_add_force; took: 0:00:00.000010
DEBUG: Trying rule: git_bisect_usage; took: 0:00:00.000010
DEBUG: Trying rule: git_branch_delete; took: 0:00:00.000009
DEBUG: Trying rule: git_branch_delete_checked_out; took: 0:00:00.000010
DEBUG: Trying rule: git_branch_exists; took: 0:00:00.000010
DEBUG: Trying rule: git_branch_list; took: 0:00:00.000009
DEBUG: Trying rule: git_checkout; took: 0:00:00.000009
DEBUG: Trying rule: git_commit_amend; took: 0:00:00.000009
DEBUG: Trying rule: git_commit_reset; took: 0:00:00.000009
DEBUG: Trying rule: git_diff_no_index; took: 0:00:00.000010
DEBUG: Trying rule: git_diff_staged; took: 0:00:00.000010
DEBUG: Trying rule: git_fix_stash; took: 0:00:00.000009
DEBUG: Trying rule: git_flag_after_filename; took: 0:00:00.000010
DEBUG: Trying rule: git_help_aliased; took: 0:00:00.000009
DEBUG: Trying rule: git_merge; took: 0:00:00.000009
DEBUG: Trying rule: git_merge_unrelated; took: 0:00:00.000009
DEBUG: Trying rule: git_not_command; took: 0:00:00.000010
DEBUG: Trying rule: git_pull; took: 0:00:00.000009
DEBUG: Trying rule: git_pull_clone; took: 0:00:00.000009
DEBUG: Trying rule: git_pull_uncommitted_changes; took: 0:00:00.000009
DEBUG: Trying rule: git_push; took: 0:00:00.000009
DEBUG: Trying rule: git_push_different_branch_names; took: 0:00:00.000039
DEBUG: Trying rule: git_push_pull; took: 0:00:00.000010
DEBUG: Trying rule: git_push_without_commits; took: 0:00:00.000009
DEBUG: Trying rule: git_rebase_merge_dir; took: 0:00:00.000009
DEBUG: Trying rule: git_rebase_no_changes; took: 0:00:00.000010
DEBUG: Trying rule: git_remote_delete; took: 0:00:00.000009
DEBUG: Trying rule: git_remote_seturl_add; took: 0:00:00.000010
DEBUG: Trying rule: git_rm_local_modifications; took: 0:00:00.000010
DEBUG: Trying rule: git_rm_recursive; took: 0:00:00.000010
DEBUG: Trying rule: git_rm_staged; took: 0:00:00.000009
DEBUG: Trying rule: git_stash; took: 0:00:00.000010
DEBUG: Trying rule: git_tag_force; took: 0:00:00.000009
DEBUG: Trying rule: git_two_dashes; took: 0:00:00.000010
DEBUG: Trying rule: go_run; took: 0:00:00.000013
DEBUG: Trying rule: go_unknown_command; took: 0:00:00.000010
DEBUG: Trying rule: gradle_no_task; took: 0:00:00.000012
DEBUG: Trying rule: gradle_wrapper; took: 0:00:00.000012
DEBUG: Trying rule: grep_arguments_order; took: 0:00:00.000012
DEBUG: Trying rule: grep_recursive; took: 0:00:00.000015
DEBUG: Trying rule: grunt_task_not_found; took: 0:00:00.000012
DEBUG: Trying rule: gulp_not_task; took: 0:00:00.000012
DEBUG: Trying rule: has_exists_script; took: 0:00:00.000013
DEBUG: Trying rule: heroku_multiple_apps; took: 0:00:00.000012
DEBUG: Trying rule: heroku_not_command; took: 0:00:00.000010
DEBUG: Trying rule: hostscli; took: 0:00:00.000013
DEBUG: Trying rule: ifconfig_device_not_found; took: 0:00:00.000011
DEBUG: Trying rule: java; took: 0:00:00.000016
DEBUG: Trying rule: javac; took: 0:00:00.000012
DEBUG: Trying rule: lein_not_task; took: 0:00:00.000013
DEBUG: Trying rule: ln_no_hard_link; took: 0:00:00.000003
DEBUG: Trying rule: ln_s_order; took: 0:00:00.000003
DEBUG: Trying rule: ls_all; took: 0:00:00.000011
DEBUG: Trying rule: ls_lah; took: 0:00:00.000013
DEBUG: Trying rule: man; took: 0:00:00.000012
DEBUG: Trying rule: mercurial; took: 0:00:00.000012
DEBUG: Trying rule: mkdir_p; took: 0:00:00.000004
DEBUG: Trying rule: mvn_no_command; took: 0:00:00.000012
DEBUG: Trying rule: mvn_unknown_lifecycle_phase; took: 0:00:00.000010
DEBUG: Trying rule: no_such_file; took: 0:00:00.000590
DEBUG: Trying rule: npm_missing_script; took: 0:00:00.000016
DEBUG: Trying rule: npm_run_script; took: 0:00:00.000011
DEBUG: Trying rule: npm_wrong_command; took: 0:00:00.000012
DEBUG: Trying rule: open; took: 0:00:00.000013
DEBUG: Trying rule: php_s; took: 0:00:00.000013
DEBUG: Trying rule: pip_install; took: 0:00:00.000012
DEBUG: Trying rule: pip_unknown_command; took: 0:00:00.000014
DEBUG: Trying rule: port_already_in_use; took: 0:00:00.000456
DEBUG: Trying rule: prove_recursively; took: 0:00:00.000013
DEBUG: Trying rule: pyenv_no_such_command; took: 0:00:00.000012
DEBUG: Trying rule: python_command; took: 0:00:00.000004
DEBUG: Trying rule: python_execute; took: 0:00:00.000011
DEBUG: Trying rule: quotation_marks; took: 0:00:00.000002
DEBUG: Trying rule: react_native_command_unrecognized; took: 0:00:00.000011
DEBUG: Trying rule: remove_shell_prompt_literal; took: 0:00:00.000002
DEBUG: Trying rule: remove_trailing_cedilla; took: 0:00:00.000002
DEBUG: Trying rule: rm_dir; took: 0:00:00.000003
DEBUG: Trying rule: scm_correction; took: 0:00:00.000012
DEBUG: Trying rule: sed_unterminated_s; took: 0:00:00.000011
DEBUG: Trying rule: sl_ls; took: 0:00:00.000002
DEBUG: Trying rule: ssh_known_hosts; took: 0:00:00.000012
DEBUG: Trying rule: sudo; took: 0:00:00.000011
DEBUG: Trying rule: sudo_command_from_user_path; took: 0:00:00.000012
DEBUG: Trying rule: switch_lang; took: 0:00:00.000002
DEBUG: Trying rule: systemctl; took: 0:00:00.000015
DEBUG: Trying rule: terraform_init; took: 0:00:00.000011
DEBUG: Trying rule: tmux; took: 0:00:00.000012
DEBUG: Trying rule: touch; took: 0:00:00.000011
DEBUG: Trying rule: tsuru_login; took: 0:00:00.000010
DEBUG: Trying rule: tsuru_not_command; took: 0:00:00.000009
DEBUG: Trying rule: unknown_command; took: 0:00:00.000165
DEBUG: Trying rule: unsudo; took: 0:00:00.000002
DEBUG: Trying rule: vagrant_up; took: 0:00:00.000013
DEBUG: Trying rule: whois; took: 0:00:00.000012
DEBUG: Trying rule: workon_doesnt_exists; took: 0:00:00.000011
DEBUG: Trying rule: yarn_alias; took: 0:00:00.000012
DEBUG: Trying rule: yarn_command_not_found; took: 0:00:00.000010
DEBUG: Trying rule: yarn_command_replaced; took: 0:00:00.000010
DEBUG: Trying rule: yarn_help; took: 0:00:00.000012
DEBUG: Trying rule: man_no_space; took: 0:00:00.000003
DEBUG: Trying rule: no_command; took: 0:00:00.000079
DEBUG: Trying rule: missing_space_before_subcommand; took: 0:00:00.000055
DEBUG: Trying rule: long_form_help; took: 0:00:00.000243
DEBUG: Trying rule: history; took: 0:00:00.008103
Nothing found
DEBUG: Total took: 0:00:00.223141
```

If the bug only appears with a specific application, the output of that application and its version:

    N/A

Anything else you think is relevant:

N/A

### PR DESCRIPTION
**Pull request by divykj**
Fix #1123

### CODE REVIEW

File: thefuck/rules/composer_not_command.py [hunk 1]
```diff
@@ -5,12 +5,19 @@
...
 
 def get_new_command(command):
-    broken_cmd = re.findall(r"Command \"([^']*)\" is not defined", command.output)[0]
-    new_cmd = re.findall(r'Did you mean this\?[^\n]*\n\s*([^\n]*)', command.output)
-    if not new_cmd:
-        new_cmd = re.findall(r'Did you mean one of these\?[^\n]*\n\s*([^\n]*)', command.output)
-    return replace_argument(command.script, broken_cmd, new_cmd[0].strip())
+    if "install" in command.script_parts and "composer require" in command.output.lower():
+        broken_cmd = "install"
+        new_cmd = "require"
```
Comment (scorphus) [inline thefuck/rules/composer_not_command.py:16]: [Nit](https://google.github.io/eng-practices/review/reviewer/standard.html):

```suggestion
        broken_cmd, new_cmd = "install", "require"
```

**Review by scorphus — APPROVED** [review]
LGTM :+1:

Thnks for contributing!

File: thefuck/rules/composer_not_command.py (hunk 1 above)
Comment (divykj) [reply thefuck/rules/composer_not_command.py]: Done 🙌

Comment (scorphus) [reply thefuck/rules/composer_not_command.py]: 🤜🤛

**Comment by scorphus** [comment]
Thanks, @divykj! 🤙

**Pull request merged**